File: web2-project/src/app/model/address.ts
export class Address {
    public addressID: number;
    public street: string;
    public number: number;
    public city: string;
    public postalNumber: number;
    public priority: number;

    constructor(addressID: number, street: string, number: number, city : string, postalNumber : number, priority: number){
        this.addressID = addressID;
        this.street = street;
        this.number = number;
        this.city = city;
        this.postalNumber = postalNumber;
        this.priority = priority;
    }
}

File: Web2BackEnd/Web2BackEnd/Models/Address.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Web2BackEnd.Models
{
	public class Address : GenericModel
	{
		[Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
		[ForeignKey("Consumer")]
		public int AddressID { get; set; }
		[Required]
		public string Street { get; set; }
		[Required]
		public int Number { get; set; }
		[Required]
		public string City { get; set; }
		[Required]
		public int PostalNumber { get; set; }
		[Required]
		public int Priority { get; set; }
	}
}
